proto/identity/api/v1/identity.proto

```diff
@@ -137,3 +145,27 @@ message GetInboxIdsResponse {
 
   repeated Response responses = 1;
 }
+
+// Request to subscribe to association changes, triggered by Identity Updates
+message SubscribeAssociationChangesRequest {}
+
+message SubscribeAssociationChangesResponse {
```

Comment:
Is there any world where this can just be a stream of identity updates, with the parsing of the stream done client-side? Would be simpler/more compatible with decentralization/requiring less server trust

Reply:
I'm hoping in a decentralized world we can create a subgraph that does that aggregation by reading events from the blockchain and running our code. Then we can slice and dice the data any way we want.

But for now, it's a big ask for clients to run `libxmtp` from whatever language/framework they are using and correctly store previous association states and apply updates.